shared/constants/crypto.tsx

```diff
@@ -30,7 +30,7 @@ export const Tabs: Array<Types.Tab> = [
     title: TabTitles[signTab],
   },
   {
-    icon: 'iconfont-question-mark',
+    icon: 'iconfont-verify',
```

Comment:
@jacobhaven I had to regenerate the icon font because we already had an icon named "iconfont-question-mark". But let's chat tomorrow - this new icon isn't showing up.